plugin/compile-scss.js

```diff
@@ -4,38 +4,16 @@ const path = Plugin.path;
 const fs = Plugin.fs;
 
 const compileSass = promisify(sass.render);
-let _includePaths = [];
 const rootDir = (process.env.PWD || process.cwd()) + "/";
 
+const { includePaths } = _getConfig('scss-config.json');
+const _includePaths = Array.isArray(includePaths) ? includePaths : [];
```

Comment:
This is now just loading the config one time and setting _includePaths right away. Should fix all problems from both #306 and #289.

Reply:
Also, this should be safe to do because `_getConfig` will return an empty object if no configuration is found.